thingooConnector/mqttconnector.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+import paho.mqtt.client as mqtt
+
+from thingooConnector.connector import Connector
+
+logger = logging.getLogger(__name__)
+
+
+class Userdata:
```

Comment:
I would rename this class to ClientCredentials as in http Connector

Reply:
If we have HTTPConnector and MQTTConnector etc. (with uppercase)
It will be probably better to name this class MQTTCredentials.

Reply:
Agreed

Reply:
Resloved in aabc133